core_impl/src/main/java/io/opencensus/stats/MutableViewData.java

```diff
@@ -61,6 +72,9 @@
       Maps.newHashMap();
   @Nullable private final Timestamp start;
 
+  // The queue of TimestampedValues for interval aggregations.
+  private final Queue<TimestampedValue> valueQueue = new LinkedList<TimestampedValue>();
```

Comment:
I think we need to also record the tag values for each value, so we know which aggregations they are part of.  We could either use one queue per tag value combination or add the tag values to the elements of the queue.